README.md

```diff
@@ -91,10 +59,13 @@ PORT=3001 REACT_APP_FM_GATEWAY_API="http://127.0.0.1:8175" REACT_APP_FM_GATEWAY_
 ```
 
 ### Run with Docker
+
 **Note:** Docker images are only built for `linux/amd64`. Your docker will need to support virtualization to run on other platforms.
 
 #### Guardian UI
+
 The guardian UI container is available at [`fedimintui/guardian-ui`](https://hub.docker.com/r/fedimintui/guardian-ui)
+
 ```
```

Comment:
oh this should be guardian ui docker run

Reply:
Whew, good catch!